includes/class-admin-settings.php

```diff
@@ -110,14 +113,14 @@ public function reset_settings() {
 	 */
 	public function reset_admin_notice() {
 		if (
-			( 'true' === get_option( 'aspireupdate-reset' ) ) &&
+			( 'true' === get_site_option( 'aspireupdate-reset' ) ) &&
 			isset( $_GET['reset-success'] ) &&
 			( 'success' === $_GET['reset-success'] ) &&
 			isset( $_GET['reset-success-nonce'] ) &&
 			wp_verify_nonce( sanitize_key( $_GET['reset-success-nonce'] ), 'aspireupdate-reset-success-nonce' )
 		) {
-			echo '<div class="notice notice-success is-dismissible"><p>' . esc_html__( 'Settings have been reset to default.', 'AspireUpdate' ) . '</p></div>';
```

Comment:
I don't know how the i18n and escaping got removed. Sorry.

Reply:
I assume this made it into the PR?

Reply:
It didn't but I made another PR for it.